@ndlib/gatsby-theme-marble/src/components/Shared/ManifestCard/ManifestCardChildren/index.js

```diff
@@ -4,14 +4,15 @@ import React from 'react'
 import PropTypes from 'prop-types'
 import { jsx } from 'theme-ui'
 import ChildField from './ChildField'
+import typy from 'typy'
 
 export const ManifestCardChildren = ({ parentProps, date, creator, collectionName }) => {
-  const collectionDisplay = 'Part of: ' + collectionName
+  const collectionDisplay = 'Part of ' + collectionName
   return (
     <>
       <ChildField field={creator} />
       <ChildField field={date} />
-      <ChildField field={collectionName ? collectionDisplay : ''} />
+      <ChildField field={typy(collectionName).safeArray.length > 0 ? collectionDisplay : ''} />
```

Comment:
This logic looks fishy. `collectionName` sounds more like a string than an array... But if it is supposed to be an array, then you can't simply concatenate it to a string when setting `collectionDisplay`

Reply:
Fixed to check first array item's string value